packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -247,8 +248,8 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
         library: 'rendering library',
         context: ErrorDescription('during layout'),
         renderObject: this,
-        offendingContext: debugCreator,
         informationCollector: () sync* {
+          yield DiagnosticsProperty<Object>('offendingContext', debugCreator, level: DiagnosticLevel.hidden);
```

Comment:
this property should be just called 
'debugCreator' as that is what it is.
The 'offendingContext' is something the transform later derives from it.

Reply:
fix here and elsewhere.

Reply:
Make this property DiagnosticsProperty<DebugCreator> instead of DiagnosticsProperty<Object> so you can detect the offendingContext by type instead of String.

Reply:
At this layer we don't know DebugCreator.